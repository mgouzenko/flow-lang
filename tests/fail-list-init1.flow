// Should not be able to put tail of char list into the initialization of int list 

int main() {
  list <int> test = [1, 2, 3, 4, 5];
  int head = @test;
  print_int(head);
  println();
  list <char> tail = ^test; // FAIL
}