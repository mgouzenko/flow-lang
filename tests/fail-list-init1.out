Fatal error: exception Failure("tail: Declaration type does not match expression")